march_rqt_gait_generator/test/nosetests/gait_generator_controller_test.py

```diff
@@ -0,0 +1,347 @@
+#!/usr/bin/env python
```

Comment:
This is not necessary since the file should not be executable. You also have to do

    chmod -x  march_rqt_gait_generator/test/nosetests/gait_generator_controller_test.py